clients/python-wrapper/lakefs/__main__.py

```diff
@@ -151,20 +151,19 @@ def _find_binary(binary_name: str) -> Optional[str]:
     return binary_path
 
 
-def run_binary(binary_path: str, args: list[str]):
+def run_binary(binary_path: str, args: list[str]) -> NoReturn:
     '''
     Run the binary with the provided arguments.
     '''
-    if not binary_path:
-        raise RuntimeError("binary not found")
-    binary_name = os.path.basename(binary_path)
-    try:
-        print(f'running {binary_path}...')
-        proc = subprocess.run(
-            [binary_path] + args, check=False, env=os.environ)
-        return proc.returncode
-    except subprocess.CalledProcessError as e:
-        raise RuntimeError(f"Error executing {binary_name}: {e}") from e
+    print(f'running {binary_path}...', flush=True)
+    if sys.platform == 'win32':
+        try:
+            proc = subprocess.run([binary_path, *args], check=False)
+        except KeyboardInterrupt:
+            sys.exit(1)
+        sys.exit(proc.returncode)
```

Comment:
Poor Windows users.  But since it's unclear how to improve matters, and since the existing experience is certainly more than good enough, I see no reason to add work here.